### PR TITLE
chore(release): migration script worker visibility

### PR DESCRIPTION
**Pull request by timhuynh94**
Ref: https://github.com/go-vela/server/pull/772

### CODE REVIEW

File: migrations/v0.20/alter.go [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+// database to modify the tables that require new or
+// modified columns.
+func (d *db) Alter() error {
+	logrus.Debug("executing alter from provided configuration")
+
+	logrus.Infof("altering %s table to add visibility columns", constants.TableWorker)
+	// alter builds table to add event_action column
+	err := d.Gorm.Exec(AlterWorkers).Error
+	if err != nil {
+		return fmt.Errorf("unable to alter %s table: %v", constants.TableRepo, err)
```
Comment (ecrupper) [inline migrations/v0.20/alter.go:37]:
```suggestion
		return fmt.Errorf("unable to alter %s table: %v", constants.TableWorker, err)
```

File: migrations/v0.20/main.go [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+)
+
+func main() {
+	app := cli.NewApp()
+
+	// Utility Information
+
+	app.Name = "vela-migration"
+	app.HelpName = "vela-migration"
+	app.Usage = "Vela utility used to migrate from v0.18.x to v0.19.x"
```
Comment (ecrupper) [inline migrations/v0.20/main.go:25]:
```suggestion
	app.Usage = "Vela utility used to migrate from v0.19.x to v0.20.x"
```

File: migrations/v0.20/main.go [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+		&cli.BoolFlag{
+			EnvVars: []string{"VELA_ACTION_ALL", "ACTION_ALL"},
+			Name:    "action.all",
+			Usage:   "enables running all actions for v0.19.x",
+		},
+		&cli.BoolFlag{
+			EnvVars: []string{"VELA_ALTER_TABLES", "ALTER_TABLES"},
+			Name:    "alter.tables",
+			Usage:   "enables altering the table configuration for v0.19.x",
```
Comment (ecrupper) [inline migrations/v0.20/main.go:54]:
```suggestion
			Usage:   "enables altering the table configuration for v0.20.x",
```

**Review by ecrupper — CHANGES_REQUESTED** [review]
Pesky v19 related content

**Review by ecrupper — APPROVED** [review]

**Review by plyr4 — APPROVED** [review]

**Pull request merged**